index.html

```diff
@@ -160,99 +160,102 @@ <h2 class="title">Tonic</h2>
           <ul class="skills">
             <li>html</li>
             <li>css</li>
+            <li>Ruby on rails</li>
             <li>javascript</li>
           </ul>
           <button class="button"> See project </button>
          </div>
         </div>
     </section>
     <section class="about_myself">
-      <div class="intro top">
-        <h2 class="pb branding">
-          About 
-          <br />
-         Myself
-        </h2>
-        <p class="pb introduction">
-         Hello I am a software developer! I can help you build a product, feature or
-          website look through some of your work and experience! If you like
-          what you see and have a project you need coded, don't hesitate to
-          contact me.
-        </p>
-        <a href="#" class="pb connect">LET'S CONNECT</a>
-          <ul class="pb">
-          <li>
-            <a class="icons" href="mailto:sawmon71293@gmail.com">
-              <ion-icon name="mail-outline"></ion-icon>
-            </a>
-          </li>
-          <li>
-            <a class="icons" href="https://twitter.com/sawmon34268255">
-              <ion-icon name="logo-twitter"></ion-icon>
-            </a>
-          </li>
-          <li>
-            <a class="icons" href="https://github.com/sawmon71293">
-              <ion-icon name="logo-github"></ion-icon>
-            </a>
-          </li>
-          <li>
-            <a class="icons" href="https://www.linkedin.com/in/saw-mon-han/">
-              <ion-icon name="logo-linkedin"></ion-icon>
-            </a>
-          </li>
-          </ul>
-        <button class="button button1">Get my resume</button>
-        <ul class="showcase display-flex column">
-          <li class="mt-45">
-            <div class="display-flex">
-               <span>Language</span>
-              <ion-icon class="arrow" name="chevron-down-outline"></ion-icon>
-            </div>
-            <ul class="language display-flex column">
-              <li class="lang">
-                <img class="w-48" src="img/js.png" alt="Javascript">
-                <span>Javascript</span></li>
-              <li class="lang">
-                <img class="w-48" src="img/html.png" alt="HTML">
-                <span>HTML</span>
-              </li>
-              <li class="lang">
-                <img class="w-48" src="img/css.png" alt="CSS">
-                <span>CSS</span>
-              </li> 
+      <div class="intro top desktop-intro">
+        <div class="ml-25">
+          <h2 class="pb branding">
+            About
+            <br />
+           Myself
+          </h2>
+          <p class="pb introduction">
+           Hello I am a software developer! I can help you build a product, feature or
+            website look through some of your work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </p>
```

Comment:
- [ ] You did a good job styling your About me section :+1: :100:, but to improve the user experience I think it would be better if you consider fixing errors in screen sizes less than 1250px because it is a requirement to desktop version should be fully responsive. Below you can see some errors on screens less than 1250px

![Screenshot from 2023-01-25 05-13-46](https://user-images.githubusercontent.com/105117832/214538202-8af6cf9f-cd91-4170-8a58-25d7ae179b7b.png)

![Screenshot from 2023-01-25 05-21-11](https://user-images.githubusercontent.com/105117832/214538417-f54058aa-ad87-42f7-aea7-f1b719fbb866.png)

![Screenshot from 2023-01-25 05-21-36](https://user-images.githubusercontent.com/105117832/214538459-1aad9e1d-9643-4a00-b4b2-e2ee11993d03.png)